ci-operator/step-registry/ipi/conf/nutanix/external-lb/ipi-conf-nutanix-external-lb-ref.yaml

```diff
@@ -0,0 +1,16 @@
+ref:
+  as: ipi-conf-nutanix-external-lb
+  from: upi-installer
```

Comment:
If this step does not care about installer version, suggest to use a specific upi-installer version, e.g:
```
  from_image:
   namespace: ocp
   name: "4.14"
   tag: upi-installer
```